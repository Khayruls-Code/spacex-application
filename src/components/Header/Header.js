import React from 'react';
import img from '../../images/logo.png'
const Header = () => {
  return (
    <div className='bg-white w-full py-3 mb-6'>
      <div className='container px-4 mx-auto flex items-center sm:justify-between justify-center flex-col sm:flex-row'>
        <div className='flex items-center'>
          <img className='w-16' src={img} alt="logo" />
          <h1 className='font-bold text-xl text-transparent font-san'>SpaceX Dashboard</h1>
        </div>
        <ul className='flex items-center gap-4'>
          <li><a className='text-md font-medium text-transparent' href="#">Rockets</a></li>
          <li><a className='text-md font-medium text-transparent' href="#">Missions</a></li>
          <li><a className='text-md font-medium text-transparent' href="#">Profile</a></li>
        </ul>
      </div>
    </div>
  );
};

export default Header;