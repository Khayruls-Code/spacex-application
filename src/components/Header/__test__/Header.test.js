import React from 'react'
import ReactDOM from 'react-dom'
import Header from '../Header'
import renderer from 'react-test-renderer'


it('Header render without crashing',
() => {
  const div = document.createElement('div')
  ReactDOM.render(<Header />,
div)
})

it("matchs Header snapshot",
() => {
  const tree = renderer.create(<Header />).toJSON()
  expect(tree).toMatchSnapshot()
})