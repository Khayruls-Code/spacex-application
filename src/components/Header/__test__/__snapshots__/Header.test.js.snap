// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matchs Header snapshot 1`] = `
<div
  className="bg-white w-full py-3 mb-6"
>
  <div
    className="container px-4 mx-auto flex items-center sm:justify-between justify-center flex-col sm:flex-row"
  >
    <div
      className="flex items-center"
    >
      <img
        alt="logo"
        className="w-16"
        src="logo.png"
      />
      <h1
        className="font-bold text-xl text-transparent font-san"
      >
        SpaceX Dashboard
      </h1>
    </div>
    <ul
      className="flex items-center gap-4"
    >
      <li>
        <a
          className="text-md font-medium text-transparent"
          href="#"
        >
          Rockets
        </a>
      </li>
      <li>
        <a
          className="text-md font-medium text-transparent"
          href="#"
        >
          Missions
        </a>
      </li>
      <li>
        <a
          className="text-md font-medium text-transparent"
          href="#"
        >
          Profile
        </a>
      </li>
    </ul>
  </div>
</div>
`;
